backend/spec/controllers/spree/admin/orders_controller_spec.rb

```diff
@@ -296,19 +296,12 @@
     end
 
     context "#open_adjustments" do
-      let(:closed) { double('closed_adjustments') }
-      let(:closed_adjustment) { double }
-
-      before do
-        allow(adjustments).to receive(:where).and_return(closed)
-        allow(closed).to receive(:each).and_return([])
-      end
+      let(:order) { create(:order) }
+      let!(:closed_adjustment) { create(:adjustment, finalized: true, adjustable: order, order: order) }
```

Comment:
Hm? I'm confused. Now that the adjustments aren't called "open" anymore but "finalized" it seemed more straightforward to call it that. I was just suggesting a rename of the `let!` from `:closed_adjustment` to `finalized_adjustment` for consistency. It seems like that's only used locally so shouldn't be a lot of work?

But! Totally :+1: for merging this as is, this is just me nitpicking.